### ISSUE TITLE
Reading Log Feature /accounts/my-books page broken on dev local

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#719]
See related: https://github.com/internetarchive/openlibrary/issues/693
Reading Log feature requires additional tables to be built (which are currently in the schema) -- see:

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/core/schema.py

**Comment by mekarpeles** [comment]
Interim fix is to add the following tables to postgres via vagrant ssh:

```
CREATE TABLE bookshelves (
id serial not null primary key,
name text,
description text default null,
archived BOOLEAN DEFAULT FALSE,
updated timestamp without time zone default (current_timestamp at time zone 'utc'),
created timestamp without time zone default (current_timestamp at time zone 'utc')
);

CREATE TABLE bookshelves_books (
username text NOT NULL,
work_id integer NOT NULL,
bookshelf_id INTEGER references bookshelves(id) ON DELETE CASCADE ON UPDATE CASCADE,
edition_id integer default null,
updated timestamp without time zone default (current_timestamp at time zone 'utc'),
created timestamp without time zone default (current_timestamp at time zone 'utc'),
primary key (username, work_id, bookshelf_id)
);

CREATE TABLE bookshelves_votes (
username text NOT NULL,
bookshelf_id serial NOT NULL REFERENCES bookshelves(id) ON DELETE CASCADE ON UPDATE CASCADE,
updated timestamp without time zone default (current_timestamp at time zone 'utc'),
created timestamp without time zone default (current_timestamp at time zone 'utc'),
primary key (username, bookshelf_id)
);
```

**Comment by mekarpeles** [comment]
https://github.com/internetarchive/openlibrary/blob/a04637b01ab4109217d8ed143325916ea1d3502c/scripts/bootstrap.sh#L46-L66

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/core/schema.py#L42 -- L42 to L87 are the only line's I'm confident need to be run by the bootstrap script. Also, the `bookshelves_books` and `bookshelves` table definitions need to be swapped

cc: @diamondman

**Comment by mekarpeles** [comment]
https://github.com/internetarchive/openlibrary/blob/a04637b01ab4109217d8ed143325916ea1d3502c/scripts/bootstrap.sh#L46-L66

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/core/schema.py#L42-L87

are the only line's I'm confident need to be run by the bootstrap script. Also, the `bookshelves_books` and `bookshelves` table definitions need to be swapped

cc: @diamondman

**Comment by mekarpeles** [comment]
`hotfix/working-dev-env` is a [WIP] branch which resolves #693 and #719

**Comment by mekarpeles** [comment]
resolved by #748
